### PR TITLE
Add Rush installation stuff to the build image script

### PR DESCRIPTION
**Pull request by fgalan**
The Redis+Rush installation fragment has been tested isolatedly. Thus, it would be needed a test of the whole build-fiwarelab-image.sh script after LGTM but before merging.

### CODE REVIEW

File: scripts/build/fiwarelab/build-fiwarelab-image.sh [hunk 1]
```diff
@@ -138,5 +150,209 @@ sudo iptables -I INPUT 1 -p tcp -m state --state NEW -m tcp --dport 1026 -j ACCE
...
-# TBD
+wget https://github.com/telefonicaid/Rush/archive/1.8.3.tar.gz
+tar xfvz 1.8.3.tar.gz
+sudo mkdir -p /opt
+sudo mv Rush-1.8.3/ /opt/Rush
+rm -rf 1.8.3.tar.gz
+
+RUSH_INIT=$(mktemp)
+cat > $RUSH_INIT <<'EOF'
+#!/bin/bash
```
Comment (fgalan) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: I understand that the Rush init script is not available in the Rush repository, thus I'm using a here doc based on the init script currently in orion-pab-image-R4.2.

Comment (dmoranj) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: No, it's not. All the release-related scripts and information (including Puppets and other stuff) were published in the internal git repository.

Comment (fgalan) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: NTC

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**